src/presentations/indexable-post-type-presentation.php

```diff
@@ -28,12 +29,14 @@ class Indexable_Post_Type_Presentation extends Indexable_Presentation {
 	/**
 	 * Indexable_Post_Type_Presentation constructor.
 	 *
-	 * @param Post_Type_Helper $post_type The post type helper.
-	 * @param User_Helper      $user             The user helper.
+	 * @param Post_Type_Helper $post_type  The post type helper.
+	 * @param User_Helper      $user       The user helper.
+	 * @param Date_Helper      $date       The date helper.
 	 */
-	public function __construct( Post_Type_Helper $post_type, User_Helper $user ) {
+	public function __construct( Post_Type_Helper $post_type, User_Helper $user, Date_Helper $date ) {
 		$this->post_type = $post_type;
 		$this->user      = $user;
+		$this->date      = $date;
```

Comment:
In the other files in this PR you use `$date_helper` rather than `$date`.

Reply:
I went for consistency within files for now. Making it consistent across the entire src folder is on the to do list in the overview issue.